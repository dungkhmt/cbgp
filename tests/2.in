50 50
50 125
10 5
14 5
31 7
5 12
18 13
11 44
1 18
31 43
43 5
46 14
21 13
1 8
16 12
20 50
43 1
14 36
27 4
2 1
24 48
12 12
11 4
15 8
13 29
4 1
32 33
43 20
40 41
12 26
34 10
45 30
19 19
11 50
4 35
8 18
3 20
27 6
6 32
16 17
4 6
5 7
7 14
48 44
40 14
13 13
1 12
5 3
2 14
2 8
1 30
44 43
31 39
30 18
8 23
9 15
40 46
43 49
1 13
15 4
12 17
14 33
5 20
9 19
30 30
15 23
31 47
7 7
49 4
25 50
7 39
24 6
1 19
5 1
9 42
17 26
47 43
2 30
6 8
17 17
2 12
12 36
2 36
6 2
41 7
19 18
44 33
9 8
10 3
32 2
18 42
43 7
1 3
43 50
10 20
5 34
9 4
7 28
40 7
37 17
15 27
43 4
20 25
24 38
46 7
19 22
1 20
9 18
