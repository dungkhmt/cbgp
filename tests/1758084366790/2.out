(29, 15), (19, 0), (5, 7), (23, 12), (31, 43), (25, 26), (43, 27), (19, 2), (8, 10), (8, 12), (24, 1), (9, 22), (36, 12), (0, 28), (10, 11), (17, 34), (9, 10), (30, 25), (39, 35), (10, 20), (40, 0), (40, 44), (23, 26), (9, 46), (2, 4), (29, 38), (48, 8), (0, 1), (2, 45), (29, 3), (32, 48), (2, 8), (2, 26), (4, 35), (17, 49), (9, 43), (19, 8), (5, 3), (4, 39), (10, 18), (0, 5), (4, 14), (8, 32), (21, 1), (13, 4), (49, 10), (29, 40), (25, 45), (12, 26), (12, 30), (3, 0), (45, 26), (20, 28), (17, 20), (39, 42), (46, 1), (1, 10), (17, 0), (22, 2), (20, 49), (24, 9), (22, 39), (48, 18), (47, 10), (36, 25), (36, 26), (35, 9), (23, 8), (9, 2), (1, 17), (11, 3), (10, 48), (40, 7), (29, 33), (17, 10), (2, 6), (3, 44), (32, 18), (10, 29), (38, 44), (15, 0), (10, 3), (5, 44), (38, 3), (0, 10), (41, 37), (13, 9), (9, 6), (23, 2), (34, 28), (14, 2), (34, 20), (27, 9), (8, 0), (27, 6), (15, 37), (33, 38), (0, 20), (2, 39), (8, 18), (15, 16), (27, 31), (24, 46), (25, 12), (12, 27), (9, 12), (0, 2), (28, 17), (40, 38), (1, 9), (30, 36), (3, 8), (22, 35), (42, 4), (10, 12), (42, 14), (46, 13), (18, 12), (16, 37), (47, 0), (40, 5), (15, 47), (10, 15), 
numberIntersection=1310 
minAngle=0.0 
sumAngle=14.886586562382158 
minEdgeLength=11.0 
sumEdgeLength=9103.244053055298 
minDistance=0.0 
sumDistance=362143.5394085816 

-1310.0000000000 0.0000000000 14.8865865624 11.0000000000 9103.2440530553 0.0000000000 362143.5394085816
0: (75, 100),
1: (75, 111),
2: (75, 128),
3: (66, 150),
4: (100, 100),
5: (100, 109),
6: (100, 119),
7: (100, 130),
8: (45, 150),
9: (25, 143),
10: (25, 118),
11: (100, 144),
12: (25, 105),
13: (100, 163),
14: (100, 193),
15: (25, 95),
16: (82, 200),
17: (25, 82),
18: (25, 57),
19: (45, 50),
20: (66, 50),
21: (64, 200),
22: (46, 200),
23: (27, 200),
24: (7, 200),
25: (0, 176),
26: (0, 152),
27: (0, 136),
28: (0, 124),
29: (0, 114),
30: (0, 105),
31: (0, 95),
32: (0, 86),
33: (0, 76),
34: (0, 64),
35: (0, 48),
36: (0, 24),
37: (7, 0),
38: (27, 0),
39: (46, 0),
40: (64, 0),
41: (82, 0),
42: (100, 7),
43: (100, 37),
44: (100, 56),
45: (100, 70),
46: (100, 81),
47: (75, 72),
48: (75, 89),
49: (100, 91),
75 100 75 111 75 128 66 150 100 100 100 109 100 119 100 130 45 150 25 143 25 118 100 144 25 105 100 163 100 193 25 95 82 200 25 82 25 57 45 50 66 50 64 200 46 200 27 200 7 200 0 176 0 152 0 136 0 124 0 114 0 105 0 95 0 86 0 76 0 64 0 48 0 24 7 0 27 0 46 0 64 0 82 0 100 7 100 37 100 56 100 70 100 81 75 72 75 89 100 91 0: (75, 100), 1: (75, 111), 2: (75, 128), 3: (66, 150), 4: (100, 100), 5: (100, 109), 6: (100, 119), 7: (100, 130), 8: (45, 150), 9: (25, 143), 10: (25, 118), 11: (100, 144), 12: (25, 105), 13: (100, 163), 14: (100, 193), 15: (25, 95), 16: (82, 200), 17: (25, 82), 18: (25, 57), 19: (45, 50), 20: (66, 50), 21: (64, 200), 22: (46, 200), 23: (27, 200), 24: (7, 200), 25: (0, 176), 26: (0, 152), 27: (0, 136), 28: (0, 124), 29: (0, 114), 30: (0, 105), 31: (0, 95), 32: (0, 86), 33: (0, 76), 34: (0, 64), 35: (0, 48), 36: (0, 24), 37: (7, 0), 38: (27, 0), 39: (46, 0), 40: (64, 0), 41: (82, 0), 42: (100, 7), 43: (100, 37), 44: (100, 56), 45: (100, 70), 46: (100, 81), 47: (75, 72), 48: (75, 89), 49: (100, 91), 
Time taken: 0.045 s
